### PR TITLE
Code Quality: Refactor context flyout helpers 1

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

- Renamed classes
- Format codebase

## PR Checklist

- [ ] **Development**: _N/A_
- [ ] **Approval**: Have discussed with and got approval from the team[^1]
- [x] **Tests**: Tested accessibility on the local end
- [x] **Deployment**: Deployed the app on the local end
- [ ] **Localization**: Modified en-US string resources[^2]
- [ ] **Co-authors**: _N/A_

## Screenshots

_N/A_

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.

### CODE REVIEW

File: src/Files.App/Helpers/MenuFlyout/LegacyMenuFlyoutHelper.cs [hunk 1]
```diff
@@ -12,10 +12,9 @@
 
 namespace Files.App.Helpers
 {
-	public class MenuFlyoutHelper : DependencyObject
+	[Obsolete("Must not use this helper to generate menu flyout any longer.")]
```
Comment (yaira2) [inline src/Files.App/Helpers/MenuFlyout/LegacyMenuFlyoutHelper.cs:15]: I think marking `Obsolete` is enough

**Review by yaira2 — DISMISSED** [review]

File: src/Files.App/Helpers/MenuFlyout/LegacyMenuFlyoutHelper.cs (hunk 1 above)
Comment (0x5bfa) [reply src/Files.App/Helpers/MenuFlyout/LegacyMenuFlyoutHelper.cs]: Done!

**Review by yaira2 — DISMISSED** [review]
LGTM

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 2]
```diff
@@ -262,7 +262,7 @@
 					<local:OpacityIcon Style="{StaticResource ColorIconZip}" />
 
 					<AppBarButton.Flyout>
-						<MenuFlyout helpers:MenuFlyoutHelper.IsVisible="{x:Bind ViewModel.CanExtract, Mode=OneWay}" Placement="Bottom">
+						<MenuFlyout helpers:MenuFlyoutHelper.="{x:Bind ViewModel.CanExtract, Mode=OneWay}" Placement="Bottom">
```
Comment (yaira2) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:265]:
```suggestion
						<MenuFlyout helpers:MenuFlyoutHelper.IsVisible="{x:Bind ViewModel.CanExtract, Mode=OneWay}" Placement="Bottom">
```

**Pull request merged**